packages/builder/src/components/integration/QueryViewer.svelte

```diff
@@ -44,6 +44,21 @@
   $: readQuery = query.queryVerb === "read" || query.readable
   $: queryInvalid = !query.name || (readQuery && data.length === 0)
 
+  //Cast field in query preview response to number if specified by schema
+  $: {
+    for (let i = 0; i < data.length; i++) {
+      let row = data[i]
+      for (let fieldNameIndex in Object.keys(fields)) {
```

Comment:
Just a minor comment, you can do `for (let fieldName of Object.keys(fields))` to remove the need for the next line.